Sign_Up_Demo/Controllers/ViewController.swift

```diff
@@ -16,8 +16,9 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
     }
-    
-    
+}
+
+extension ViewController {
     @IBAction func btnSignUp(_ sender: UIButton) {
```

Comment:
Add a line space after @IBAction like 
 @IBAction 
 func btnSignUp(_ sender: UIButton) {
// *Code*
}

Reply:
Also this method is not fulfilling the single responsibility principle. you should move the validation to a separate method.